### ISSUE TITLE
"Close Menu" menu-backer navigation control missing link name

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/freedom#39]
In addition to the other issues present regarding the "Close Menu" navigation control (as discussed in Issue #37), which becomes visible when the menu transforms into a drawer, this link is missing a name. Considering employing `aria-labelledby` or descriptive text here instead of a title to correct.

**Comment by aviannamiller** [comment]
For more background on this issue, please see the following automated testing result on this element from Accessibility Insights:

> Title: WCAG 4.1.2: ARIA commands must have an accessible name (.main-navigation__toggle)
> Tags: Accessibility, WCAG 4.1.2, aria-command-name
> 
> Issue: ARIA commands must have an accessible name (aria-command-name - https://dequeuniversity.com/rules/axe/4.10/aria-command-name?application=msftAI)
> 
> Target application: Search results for “” · Testing · Testing - https://dev.omeka.org/amiller/OmekaS/s/testing/index/search?fulltext_search=
> 
> Element path: .main-navigation__toggle
> 
> Snippet: `<div class="main-navigation__toggle open" tabindex="0" role="button" aria-expanded="true" aria-controls="menu-drawer"> <span></span> <span></span> <span></span> </div>`
> 
> Environment: Chrome version 137.0.0.0
> 
> ====
> 
> This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Either a title or ARIA attribute should be employed here to ensure the toggle has an accessible name.

**Comment by nelsonamaya82** [comment]
Thanks @aviannamiller , this is fixed in the master branch, please double check and let me know if it works as expected.

**Comment by aviannamiller** [comment]
The link now has an accessible name, however, I believe your updates on #37 may have already fixed this. I apologize as I may have confused things with the snippet cited in the above comment, which I did not realize to be pulling a different element than I originally opened this issue about. So, originally, `menu-backer` was missing an accessible name. That particular element appears to be fixed now via an `aria-label` (as well as the `title`). However, I noticed the collapsed nav toggle (`main-navigation__toggle`) is now announcing as "Close" via screen reader. I assume this issue is from the `aria-labelledby` attribute pointing to the `menu-backer` link. I think this comes from some confusion I caused with my original suggestion and slightly contradictory follow up, so I apologize for that! I think that particular attribute could drop off `main-navigation__toggle` at this point and be more appropriately replaced by an `aria-label` that is reflexive to the state change. Please let me know if I can better clarify anything here!

Now, for the link color, I am not seeing a change. It looks like the current text color is inheriting from the following style:
```
body,button,input,select,optgroup,textarea {
    color: #666;
    font-family: "Open Sans",sans-serif;
    font-size: 1rem;
    line-height: 1.5
}
```
The color of the arrow image on the toggle is separately informed by `filter` on `.menu-drawer .navigation-controls a::before`, so the color on that both does not match the text and is a bit under the color contrast requirements for UI components (must be at least 3:1, right now its 2.9:1 with #808080 on #DDDDDD).
